packages/sdk/src/utils.ts

```diff
@@ -319,6 +319,51 @@ export interface Criteria {
   filterData: Hex;
 }
 
+/**
+ * Whether claimaint is inferred from event or function
```

Comment:
Let's generalize the language so we can re-use this enum for stuff other than claimant